src/System.Management.Automation/resources/FileSystemProviderStrings.resx

```diff
@@ -331,12 +331,15 @@
     <value>Maximum size for drive has been exceeded: {0}.</value>
   </data>
   <data name="SymlinkItemExists" xml:space="preserve">
-    <value>Cannot create symbolic link because the path {0} already exists.</value>
+    <value>Cannot create link because the path {0} already exists.</value>
```

Comment:
It would be more easy for users to read the message if we moved the `{0}` to end.

Reply:
Good point, will update